src/biogeophys/PhotosynthesisMod.F90

```diff
@@ -1934,9 +1934,9 @@ subroutine Photosynthesis ( bounds, fn, filterp, &
 
                ! Convert gs_mol (umol H2O/m**2/s) to gs (m/s) and then to rs (s/m)
 
-               gs = gs_mol(p,iv) / cf
+               gs = gs_mol(p,iv) / cf * o3coefg(p)
                rs_z(p,iv) = min(1._r8/gs, rsmax0)
-               rs_z(p,iv) = rs_z(p,iv) / o3coefg(p)
```

Comment:
@lifang0209 explained to me that moving the effect from the resistance to the conductance, also moves it to the current time step rather than the following time step. I have not checked the validity of that statement.

However @lifang0209 you said that you had tested and found the effect of this change on the simulation to be very small. Could you post some sort of documentation here (possibly a figure) that would make this point clear?

Reply:
I have added comment in the PhotosynthesisMod.F90:
! Fang Li revised o3 impact on gs
! now: multiply gs response factor (o3coefg) with gs
! clm5.0: skipped the effect on gs and had rs divided by o3coefg,
! which led to a one timestep delay in the o3 impact on gs,
! although the impact on an annual scale is quite small